### PR TITLE
fix: babel preset + webpack config tweaks

### PR DESCRIPTION
**Pull request by zamotany**

### CODE REVIEW

File: packages/haul-babel-preset-react-native/src/index.ts [hunk 1]
```diff
@@ -9,12 +9,9 @@ const defaultPlugins = [
   // For some reason native async/await don't behave correctly
   // on RN 0.59 on both platforms, so we need to transpile it
   // to native Promises.
-  [require('babel-plugin-transform-async-to-promises')],
-  [
-    require('@babel/plugin-transform-modules-commonjs'),
-    { allowTopLevelThis: true },
-  ],
+  // [require('babel-plugin-transform-async-to-promises')],
```
Comment (thymikee) [inline packages/haul-babel-preset-react-native/src/index.ts]: the comment is left for a reason?

Comment (zamotany) [inline packages/haul-babel-preset-react-native/src/index.ts]:
```suggestion
```

Comment (zamotany) [reply packages/haul-babel-preset-react-native/src/index.ts]: nope, removed

File: packages/haul-ram-bundle-webpack-plugin/src/__tests__/integration.test.ts [hunk 2]
```diff
@@ -389,6 +389,6 @@ describe('File RAM bundle should build and successfully evaluate bundle', () =>
     );
 
     expect(mappedFrame1).toMatch('__fixtures__/nestedCrashFn.esm.js:4:9');
-    expect(mappedFrame2).toMatch('__fixtures__/crashFn.async.js:5:5');
+    expect(mappedFrame2).toMatch('__fixtures__/crashFn.async.js:5:16');
```
Comment (thymikee) [inline packages/haul-ram-bundle-webpack-plugin/src/__tests__/integration.test.ts]: is this expected?

Comment (zamotany) [reply packages/haul-ram-bundle-webpack-plugin/src/__tests__/integration.test.ts]: I think so, though I'm not 100% sure, `5` was pointing to
```
crashNested()
^
```
and `16`:
```
crashNested()
           ^
```

**Pull request merged**